composer.json

```
@@ -30,7 +30,7 @@
         "ext-intl": "*",
         "ext-json": "*",
         "ext-simplexml": "*",
-        "api-platform/core": "^3.4.2",
+        "api-platform/core": "^4.0",
```

```suggestion
        "api-platform/core": "^4.0.3",
```
to be sure, that our fixes in XmlExtractor are available in APIP